src/librawspeed/decoders/ArwDecoder.cpp

```diff
@@ -258,6 +264,78 @@ void ArwDecoder::DecodeUncompressed(const TiffIFD* raw) const {
     u.decodeRawUnpacked<16, Endianness::little>(width, height);
 }
 
+void ArwDecoder::DecodeLJpeg(const TiffIFD* raw) const {
+  uint32_t width = raw->getEntry(TiffTag::IMAGEWIDTH)->getU32();
+  uint32_t height = raw->getEntry(TiffTag::IMAGELENGTH)->getU32();
+  uint32_t bitPerPixel = raw->getEntry(TiffTag::BITSPERSAMPLE)->getU32();
+
+  switch (bitPerPixel) {
+  case 8:
+  case 12:
+  case 14:
+    break;
+  default:
+    ThrowRDE("Unexpected bits per pixel: %u", bitPerPixel);
+  }
+
+  if (width == 0 || height == 0 || height % 2 != 0 || width > 9600 ||
+      height > 6376)
+    ThrowRDE("Unexpected image dimensions found: (%u; %u)", width, height);
```

Comment:
The just released 7RM5 has

ImageWidth                      : 9728
ImageHeight                     : 6656

Reply:
@artemist Please at least bump these so ILCE-7RM5 can test as well in the meantime.

Reply:
Should be fixed now